src/adaptation/ecap/XactionRep.cc

```diff
@@ -351,7 +351,7 @@ Adaptation::Ecap::XactionRep::doneAll() const
 void
 Adaptation::Ecap::XactionRep::sinkVb(const char *reason)
 {
-    debugs(93,4, HERE << "sink for " << reason << "; status:" << status());
+    debugs(93,4, "sink for " << reason << "; status:" << status());
```

Comment:
This PR automatically removes HERE, and nothing else should be required. The spacing issue is covered in another comment, so I will not repeat those arguments here.